cmd/zdb/zdb.c

```diff
@@ -1989,7 +1989,7 @@ dump_object(objset_t *os, uint64_t object, int verbosity, int *print_header)
 	zdb_nicenum(doi.doi_max_offset, lsize);
 	zdb_nicenum(doi.doi_physical_blocks_512 << 9, asize);
 	zdb_nicenum(doi.doi_bonus_size, bonus_size);
-	zdb_nicenum(doi.doi_dnodesize, dnsize);
+	zdb_nicenum(doi.doi_dnodesize/512, dnsize);
```

Comment:
You could use DNODE_MIN_SIZE here.

Reply:
I was actually thinking I should just alter zdb to compute the unused slot count itself, rather than doing so in the test.  Then this patch wouldn't be needed.

I personally would rather see dnode slots than dnode size, but figured the output here should match the dnodesize property.  What do you think?

Reply:
That would probably be better.  Upon further reflection I think we should leave this reporting the size in bytes which is more consistent with the property.

Reply:
fixed